docs/cookbook/creating_custom_dropdowns.md

```diff
@@ -0,0 +1,98 @@
+# Creating custom dropdowns
+
+In eZ Platform you are able to implement custom dropdowns anywhere in the system.
+Follow bellow steps to learn how to manipulate small component/library in order to do so. 
```

Comment:
`Follow the steps below`, to learn how to integrate this small component to fit the project's needs`